app/views/sources/index.html.haml

```diff
@@ -2,18 +2,16 @@
 
 %h1 Sources
 
-%ul
-  - for source in sources
-    %li= source.name
-
 %table.table
   %thead
     %tr
+      %th &nbsp;
       - for type in ordered_sources.keys
         %th= type.to_s.humanize
   %tbody
-    - (0...Source.count).to_a.each do |position|
+    - (0...Source.count).to_a.each_with_index do |position, index|
```

Comment:
Found the iteration logic in this `tbody` a little hard to grok. Couple of observations:

- I think the `.to_a` is unnecessary since a `Range` such as `0...Source.count` is already an `Enumerable` and will respond to `each`

- With a Range like `0...some_num` that starts at zero, `each_with_index{ |value, index| … }` seems superflous, since the value and the index will always be identical. Would a simple `each` suffice here?